test/support/rails_application_fixture_helper.rb

```diff
@@ -0,0 +1,75 @@
+# typed: ignore
+# frozen_string_literal: true
+
+require "rails_test_helper"
+
+module RailsApplicationFixtureHelper
+  include ApplicationFixtureHelper
+
+  def setup_application_fixture
+    super()
+    cache_rails_paths
+  end
+
+  def teardown_application_fixture
+    restore_rails_paths
+    super()
+  end
+
+  def copy_template(template)
+    super(template)
+
+    case template
+    when :minimal
+      rails_autoload_paths(paths_to("/components/sales/app/models"))
+      rails_eager_load_paths([])
+      rails_autoload_once_paths([])
+      rails_root_path(app_dir)
+    when :skeleton
+      rails_autoload_paths(paths_to("components/sales/app/models"))
```

Comment:
Can we move the skeleton and minimal setup code into dedicated private methods? That will clear the method up because there is less happening in the copy_template class.